libc/include/llvm-libc-macros/limits-macros.h

```diff
@@ -0,0 +1,229 @@
+//===-- Definition of macros from limits.h --------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __LLVM_LIBC_MACROS_LIMITS_MACROS_H
+#define __LLVM_LIBC_MACROS_LIMITS_MACROS_H
+
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wgnu-include-next"
```

Comment:
What warning is this hiding? Might be able to work around it with `__has_include_next` or something.

Reply:
It is to hide the following warning from clang:
```
limits-macros.h:17:2: warning: #include_next is a language extension [-Wgnu-include-next]
```
There are the same warnings from gcc, but I'm still tracking the correct `-W` flag to ignore.

Reply:
`#pragma GCC system_header` does the trick for gcc.

Reply:
This is because include_next wasn't standardized until C23.